app/src/main/java/de/westnordost/streetcomplete/quests/building_levels/AddBuildingLevelsForm.kt

```diff
@@ -80,4 +74,47 @@ class AddBuildingLevelsForm : AbstractQuestFormAnswerFragment<BuildingLevelsAnsw
     }
 
     override fun isFormComplete() = levels.isNotEmpty()
+
+
+    private class LastPickedAdapter(
+        private val lastPickedAnswers: List<BuildingLevelsAnswer>,
+        private val onItemClicked: (position: Int) -> Unit
+    ) : RecyclerView.Adapter<LastPickedAdapter.ViewHolder>() {
+
+        class ViewHolder(
+            view: View,
+            private val onItemClicked: (position: Int) -> Unit
+        ) : RecyclerView.ViewHolder(view), View.OnClickListener {
+            val lastLevelsLabel: TextView = view.findViewById(R.id.lastLevelsLabel)
+            val lastRoofLevelsLabel: TextView = view.findViewById(R.id.lastRoofLevelsLabel)
+
+            init {
+                itemView.setOnClickListener(this)
+            }
+
+            override fun onClick(view: View) {
+                onItemClicked(bindingAdapterPosition)
+            }
```

Comment:
Why not simply something like

`itemView.setOnClickListener { _ -> onItemclicked(bindingAdapterPosition) }`

?

Reply:
Old Java habits...